lib/gui.py

```diff
@@ -127,8 +127,8 @@ def validate_input(host_address, ports_range, n_threads):
         if not re.match(regex, host_address):
             messagebox.showerror("Error", "Invalid host adress")
         elif len(ports) != 2 or\
-            (lower_port.isdigit() == False and int(lower_port) > 0) or\
-            (upper_port.isdigit() == False and int(upper_port) > 0) or \
+            (lower_port.isdigit() == False and upper_port.isdigit() == False) or\
```

Comment:
now both of them need to be not digit for this to catch it. i guess something like `asdf-10` wouldn't be caught. `and` -> `or`?